sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaSuite.scala

```diff
@@ -1040,6 +1040,14 @@ class ParquetSchemaSuite extends ParquetSchemaTest {
     }
   }
 
+  test("SPARK-40819 - ability to read parquet file with TIMESTAMP(NANOS, true)") {
```

Comment:
nit: other tests in this file follow the naming convention "SPARK-XXXXX: ..."

```suggestion
  test("SPARK-40819: ability to read parquet file with TIMESTAMP(NANOS, true)") {
```